rfcs/text/0000-zendesk-triage-sop.md

```diff
@@ -0,0 +1,150 @@
+
+3. If an answer is not evident from searching the knowledge base in the data portal, the triager needs to ask for help from the secondary support team, which includes members from each DCP component or service. It may
+be obvious which team should receive the ticket; if so, then it can be
+assigned directly. If it’s not obvious, there can be some internal
+discussion with the secondary support staff about who should receive the
+ticket.
+
+4. At this point the ticket becomes the responsibility of the secondary support person, who should repond to the user and bring the ticket to an eventual close. 
+
+
+In order to improve our User Support the triager will also work with the secondary support team to:
+
+1. tag each ticket with basic tags (data store/ingest/contribution/etc.) that help DCP staff categorize it for later retrieval. In addition, Github tickets should be made for issues like: updating docs or writing new documentation and improving our FAQs, code examples, bug reports, etc. As much as possible, issues should be turned into documentation or short FAQs and made available for users to access from the portal.
+
+2. create and maintain canned responses (how to connect/get involved with the HCA;
+how to submit data, etc.) that can be quickly sent to users who have written with those questions.
+
+3. document discussions and decisions that arise from particular tickets, so that it's clear who should respond to similar questions in the future and what the response should be.
+
+
+## Secondary Support
+
+The secondary support team is comprised of 1-2 members from each major
+component or service of the DCP. These representatives should have
+enough technical knowledge to answer most questions. Answers generated
+by the secondary support team will be added to the knowledge base. They
+may require additional information from the person who created the
+ticket. They also may need to consult with technical experts who make up
+the tertiary support teams.
+
+## Tertiary Support
+
+The Tertiary team is the entire DCP team. In the event that the
+secondary support team is not able to respond to the question, anyone
+may be called on to answer it. However, the need for Tertiary support
+should be rare, and all answers will be archived and added to the
+knowledge
+base
+
+### **Unresolved Questions**
+
+We suggest that funding be sought to hire one or two part or full-time
+triagers who would grow into the position as we move toward GA, and be
+able to handle primary triage as well as some of the more difficult
+questions by the time of the GA, when we expect support needs to
+increase significantly.
+
+> 2 triage agents - could be 50% of their job as they would trade off
+> being point for this job. Duties would include the grooming of the FAQ
+> knowledge base. They would also be responsible for moderate our user
+> forum.
+> 
+> 1 point person per box (4-5 people) - this should be at least 25% of
```

Comment:
I'm generally happy with the SOP - the updates seem very reasonable! I wanted to mention that the "at least 25% of their time" strikes me as being a bit high, and probably will depend on the component? If I were the point person (I think this is the Secondary Support team member?) for metadata or wrangling, this would be 2 hours out of my day each day, which seems a bit over kill :) I understand the desire to put something concrete here, so again, happy to merge this SOP as-is and maybe think a bit about the time commitment.

Reply:
I agree that with the current level of tickets (1 a week) 25% of 5 peoples time seems like a lot. That is 1.25FTEs committed to this on top of the 1FTE dedicated personnel

I would suggest not putting a time commitment on it with the current level of tickets but noting the PO/PL who owns zendesk to monitor and ask for a defined commitment once the ticket number gets close to 1-2 a day